index.html

```diff
@@ -242,6 +242,31 @@ <h2 class="big-text">About Myself</h2>
           <hr />
         </div>
       </section>
+      <!--section form of the portfolio cards starts-->
+    <section class="sectionf">
+      <div class="SectionfText">
+        <h2 class="frm">Contact me</h2>
+        <p class="center"> If you have application you are interested in developing,
+          a feature that you need built or a project that needs 
+          coding. I'd love to help with it</p>
+          <form action="https://formspree.io/f/meqnzbev" target="_blank" method="post">
+            <p><input id="username" type="text" class="username" name="username" maxlength="30"   placeholder="Yeremias" required>
+            </p>
+            <p>
+                <input id="email" class="email" type="email" name="email"  placeholder="natajayanj14@gmail.com" required>
+            </p> 
+            <p> 
+              <textarea name="message" placeholder="write your message here" class="textarea" maxlength="500"  id="message" cols="30" rows="10" required></textarea>
+            </p>
+            <p>
+              <input type="submit" class="form-btn" value="Get It Touch" >
+            </p>
```

Comment:
- [ ] [OPTIONAL] While this is working fine. Kindly try to use `ul` and `li` tags to structure these elements in a list instead of using `p` tags which is semantically wrong.